probaforms/models/nflow.py

```diff
@@ -5,8 +5,7 @@
 from torch.autograd import Variable
 import os
 
-
-if DEVICE:=os.environ.get('device'):
+if DEVICE := os.environ.get('device'):
```

Comment:
It is not recommended to add new models and change existing ones in one PR. It it better to separate them in different PRs.